### PR TITLE
Fix invisible checkbox in dropdown menu

### LINKED ISSUE #1593: [Bug]: Invisible checkbox in dropdown menu
### Describe the bug

It's nearly impossible to tell that the "Launch in terminal" button is a check. The only way to find out is if we click on it. I'm using Bottles from Flathub-beta with libadwaita + GTk4.

### To Reproduce

1. Open a bottle
2. Click on the arrow button next to "Run Executable"

### Package

Flatpak from Flathub

### Distribution

Fedora 36 Silverblue

### Health Check

```shell
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.12-300.fc36.x86_64
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 3862990848
    Free: 3862847488
RAM:
    MemTotal: 7.2GiB
    MemAvailable: 2.5GiB
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.33'
Bottles_envs: null
```

### Display Protocol

Wayland

### Display Backend used by Bottles

Wayland

### Additional context

image

### PR DESCRIPTION
**Pull request by axtloss**
# Description
Previously the checkbox in the "launch with terminal" prompt wasn't visible,  it now looks like this:

Screenshot from 2022-07-03 22-52-03

Fixes #1593

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. 
Provide instructions so we can reproduce.
- [x] Test A
  - Run Bottles
  - Navigate to broken dialog
  - Dialog is not broken anymore

### CODE REVIEW

**Comment by axtloss** [comment]
For some reason github added 2 more commits that aren't relevant in this pr (one of them even was merged yesterday), not sure why, hope it doesn't cause any issue

File: src/ui/details-bottle.ui [hunk 1]
```diff
@@ -102,25 +102,32 @@
...
                         <property name="text" translatable="yes">Run with Arguments…</property>
                     </object>
                 </child>
                 <child>
                     <object class="GtkCheckButton" id="check_terminal">
                         <property name="label" translatable="yes">Launch in Terminal…</property>
                         <property name="halign">start</property>
+                        <property name="margin-bottom">5</property>
+                        <property name="use-underline">True</property>
+                        <property name="use_underline">True</property>
```
Comment (mirkobrombin) [inline src/ui/details-bottle.ui:130]: I think this line is a mistake

**Comment by mirkobrombin** [comment]
Thanks for your contribution, I've just added a note

File: src/ui/details-bottle.ui (hunk 1 above)
Comment (axtloss) [reply src/ui/details-bottle.ui]: Yeah it is, I've removed it in the latest commit

**Pull request merged**